recipe/meta.yaml

```diff
@@ -1,50 +1,64 @@
-{% set version = "0.21.2" %}
-
+{% set name = "woodwork" %}
+{% set version = "0.25.1" %}
 
 package:
-  name: woodwork
+  name: {{ name }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/w/woodwork/woodwork-{{ version }}.tar.gz
-  sha256: 01eb8c7fc01240bea2329d45ee5a2b2a016c7e060199e82b5aac262efad183b8
+  url: https://pypi.io/packages/source/{{name[0]}}/{{name}}/{{name}}-{{version}}.tar.gz
+  sha256: 73cb38e09229216cb7cbbaf2e4352b34d6e40ac44bf774c17e94a9047c7e98e3
+  patches:                        # [win]
+    - remove-absolute-path.patch  # [win]
```

Comment:
Isn't this patch valid for all?

Reply:
Yes, but unnecessary. I could apply it to all if that's preferable.